yb-voyager/src/cp/yugabyted/yugabyted.go

```diff
@@ -447,12 +449,71 @@ func (cp *YugabyteD) createYugabytedMetadataTable() error {
 			status VARCHAR(30),
 			invocation_timestamp TIMESTAMPTZ,
 			PRIMARY KEY (migration_uuid, migration_phase, invocation_sequence)
-			);`, YUGABYTED_METADATA_TABLE_NAME)
+			);`, YUGABYTED_METADATA_QUALIFIED_TABLE_NAME)
 
-	return cp.executeCmdOnTarget(cmd)
+	err := cp.executeCmdOnTarget(cmd)
+	if err != nil {
+		return err
+	}
+
+	alterTableCmds := []string{
+		fmt.Sprintf(`
+        DO $$
+        BEGIN
+            IF NOT EXISTS (SELECT 1 FROM information_schema.columns
```

Comment:
@sanyamsinghal I believe there is `ALTER TABLE .. ADD COLUMN IF NOT EXISTS ...` clause as well. you can maybe directly use that. 
https://www.postgresql.org/docs/current/sql-altertable.html#:~:text=ADD%20%5B%20COLUMN%20%5D%20%5B%20IF%20NOT%20EXISTS%20%5D%20column_name%20data_type%20%5B%20COLLATE%20collation%20%5D%20%5B%20column_constraint%20%5B%20...%20%5D%20%5D

Reply:
thanks @priyanshi-yb 
As per the documentation, it is from PG 9.6 onwards.
Not mentioned in version before that, but i am not sure if its just not documented or not supported also.

Reply:
Will have to check, since we certify PG 9.x-11.x
So that means PG >= 9

Reply:
oh ok, i got a bit confused. 
Since YB uses PG11 so should not be a problem.

Reply:
done